ballerina/tests/generated_client.bal

```diff
@@ -6,6 +6,7 @@ const BUILDING = "building";
 const WORKSPACE = "workspace";
 const DEPARTMENT = "department";
 const EMPLOYEE = "employee";
+const ORDER_ITEM = "orderItem";
```

Comment:
Map keys are lowercase in the current impl.

Reply:
Updated in [2e6b2be](https://github.com/ballerina-platform/module-ballerina-persist/pull/105/commits/2e6b2be4d839a813171b8e3ef81de106fff8bd3d)